### DISCUSSION TITLE
Formily2.0正式发布准备中🎉🎉🎉

### DISCUSSION THREAD

**Discussion by janryWang** [alibaba/formily#2443] [Show and tell]
正式声明

**Formily下个版本将会发布2.0.0正式版**

后续的维护计划：
- 官网formilyjs.org切换为现有v2.formilyjs.org，原有v2.formilyjs.org保持不变，增加v1.formilyjs.org用于访问1.x文档
- 日常bugfix与小feature迭代，保证2.x不会存在Break Change，如果使用过程中存在设计上的问题，一律hold到3.0需求
- 完善共建生态体系 https://github.com/formilyjs
- 思考 QueryList/QueryForm/QueryTable 3个组件是否需要纳入桥接组件库中，具体以什么形式纳入
- 完善designable
   - 进一步完善拖拽能力
   - 重新梳理包依赖结构
   - 完善文档
   - 品牌化推广

**Comment by kaoding** [comment]
formily 2.0.0  就等你了 !

**Reply by coolbob1998** [reply]
期待

**Comment by chenshuichao** [comment]
QueryList/QueryForm/QueryTable 3个组件期待在2.1版本发布！
😄

**Comment by kaoding** [comment]
formilyjs.org 网站嵌套了。

![image](https://user-images.githubusercontent.com/18542211/142861444-b159354d-07b6-4175-b040-3d10d4c1b0f1.png)

**Reply by janryWang** [reply]
chrome applications把service worker关掉

**Comment by kaoding** [comment]
明天开始正式使用。
